src/sagemaker_containers/serializers.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json as stdjson
+import textwrap
+
+import numpy as np
+from six import BytesIO, StringIO
+
+from sagemaker_containers import content_types
+
+
+class Npy(object):
```

Comment:
Why not NpySerializer (like the doc comment)?

Reply:
The namespace is being used to reduce the ambiguity:
```
from sagemaker_containers import serializers

serializers.Npy
```

```serializers.NpySerializer is just to big```

The previous idea would be: 
```
import sagemaker_containers as smc

smc.NpySerializer

```

Following PEP 8, PEP 432, and google style code. Ideally it should be:

```import sgmkr # or any small name that we come with

sgmk.serializer.Npy
```

@winstonaws is reaching out to the python community to understand better the best solution.

We scheduled a meeting specifically for this topic next Wednesday

Reply:
or `...serializer.NpySerializer`